srtcore/core.cpp

```diff
@@ -7812,17 +7832,13 @@ void CUDT::updateSrtSndSettings()
     }
 }
 
-void CUDT::updateAfterSrtHandshake(int srt_cmd, int hsv)
+void CUDT::updateAfterSrtHandshake(int hsv)
 {
-
-    switch (srt_cmd)
-    {
-    case SRT_CMD_HSREQ:
-    case SRT_CMD_HSRSP:
-        break;
-    default:
-        return;
-    }
+    HLOGC(mglog.Debug, log << "updateAfterSrtHandshake: HS version " << hsv);
+    // This is blocked from being run in the "app reader" version because here
```

Comment:
This comment I'd insist to stay - even though it doesn't make any sense in the absence of socket groups.